templates/spago.dhall

```diff
@@ -5,7 +5,7 @@ You can edit this file as you like.
 { name =
     "my-project"
 , dependencies =
-    [ "effect", "console", "psci-support" ]
+    [ "console", "effect", "psci-support" ]
```

Comment:
Why am I changing the order in this template?

Simply to make `spago install <SOMETHING_ALREADY_IN_DEPENDENCIES>` operation idempotent.
The order is now alphabetical, so the above come (which sorts the dependencies ins `addRawDeps`) doesn't change this config if it's not necessary.

Reply:
Yep, great call 💯